studio/views.py

```diff
@@ -88,3 +93,40 @@ def get(self, request, format=None):
             "auth": str(request.auth),
         }
         return Response(content)
+
+
+@login_required
+def profile(request):
+    # Get the user profile
+    try:
+        # Note that not all users have a user profile object
+        # such as the admin superuser
+        user_profile = UserProfile.objects.get(user_id=request.user.id)
+    except:  # noqa E722 OK here
```

Comment:
Is it though?
Can't we explicitly catch `NotFound` error or something? At least `Exception`

Reply:
I will add this